content/posts/irods-development-update-september-2022.md

```diff
@@ -0,0 +1,254 @@
+Title: iRODS Development Update: September 2022
+Date: 2022-09-30 12:00
+Author: Kory Draughn
+Slug: irods-development-update-september-2022
+Status: published
+
+
+We've been chopping a lot of wood this month, so let's jump right in!
+
+On the server side of things, work to add detached mode to the `unixfilesystem` resource plugin has begun. This is important as it allows one or more `unixfilesystem` resources to serve requests without the need for the physical object to be located on the local storage device. The `remote()` microservice has grown support for the `INST_NAME` hint. Just like the `delay()` microservice, the `remote()` microservice will now be able to target a specific rule engine plugin. Before this change, using the `remote()` microservice would result in the Rule Engine Plugin Framework attempting to execute the rule text on every configured rule engine plugin. While this would work, it resulted in a noisy log file. This change resolves that issue completely.
+
+Work to modernize the Audit Rule Engine Plugin is in motion and is going very well. This refactor aligns with the recent work regarding the ELK stack project. A lot of effort is being put into making the Audit plugin much better. If that sounds interesting, you can follow that work at [https://github.com/irods/irods_rule_engine_plugin_audit_amqp/pull/105](https://github.com/irods/irods_rule_engine_plugin_audit_amqp/pull/105).
+
+On the client side, you'll be happy to know that the C++ REST API has received some big updates. It can now be run independently of the iRODS server and iCommands. It has also been containerized! Building, running, and testing the C++ REST API is easier than ever. Please give it a try and let us know what you think.
+
+This month we have news about Metalnx. We've fixed issues with downloading data objects via tickets. We've improved the build process such that people wanting to make changes to the application can do so without running the risk of corrupting their local .m2 directory. Not only that, but configuration options have been added that allow administrators to hide AVUs based on one or more prefixes.
```

Comment:
this hasn't been released yet... could mention it will be in the upcoming 2.6.1 release... right?

Reply:
done